lib/gitlab/ci/templates/pipeline/GradleInstall4JPipeline.yml

```diff
@@ -5,14 +5,14 @@ variables:
   DEV_OR_RELEASE_REGEX: '^develop$|^[0-9]+\.[0-9]+$|^release\/.+$'
   DEFAULT_INSTALL4J_IMAGE: devsecops/install4j8:1.0.0-jdk11-slim-custom
   JDK_SELECTOR: "-PJDK=11"
-  INSTALLER_ARTIFACT_PATH:  build/installers
+  INSTALLER_ARTIFACT_PATH: build/installers
   INSTALLER_GRADLE_COMMANDS: makeAllInstallers makeAllBundles
   INSTALLER_NAME:
     value: "installers"
     description: "The name of the installer artifacts that can be downloaded after the job completes"
   RELEASE:
     value: ""
-    description: Determines what type of installer should be produced. Leave blank to produce a SNAPSHOT installer or enter 'true' to create a release installer.
+    description: Determines what type of installer should be produced. Leave blank to produce a SNAPSHOT, use "FINAL" to create a final release, or use any other value to produce a release candidate.
```

Comment:
Appreciate this change from true to FINAL. This aligns better with our Java pipeline release template.